doc/news/changes/minor/20170711MartinKronbichler-b

```diff
@@ -0,0 +1,6 @@
+Improved: The constructor of MappingQGeneric used a naive inversion of a cell
```

Comment:
You know, I think this should actually be in the "major changes" section. This definitely ought to be a topic for the next release paper.